### PR TITLE
feat: #188 add test to editable text

### PR DESCRIPTION
**Pull request by Alex89198900**
1.  Add test to editable text

### CODE REVIEW

File: mw-webapp/src/component/editableText/EditableText.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  it("shoud be works with EditableText mode number", () => {
+    cy.mount(createTestEditableText({value: EDITABLETEXTINPUT_VALUE_NUMBER, type: InputType.Number}));
+    cy.get(getDataCy(EDITABLETEXT_CY.trigger))
+      .dblclick();
+    cy.get(getDataCy(EDITABLETEXTINPUT_CY)).should("have.attr", "type", InputType.Number);
+    cy.get(getDataCy(EDITABLETEXTINPUT_CY)).should("have.value", EDITABLETEXTINPUT_VALUE_NUMBER);
+  });
+
+  it("shoud be render input or span depend on client actions", () => {
```
Comment (tritonJS826) [inline mw-webapp/src/component/editableText/EditableText.cy.tsx:73]:
```suggestion
  it("shoud render input or span depend on client actions", () => {
```

File: mw-webapp/src/component/editableText/EditableText.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+describe("EditableText component", () => {
+  it("shoud be renders when placeholder is visible", () => {
+    cy.mount(createTestEditableText({value: EDITABLETEXTINPUT_VALUE, type: InputType.Text}));
+    cy.get(getDataCy(EDITABLETEXT_CY.trigger))
+      .dblclick();
+    cy.get(getDataCy(EDITABLETEXTINPUT_CY)).should("have.attr", "placeholder", EDITABLETEXTINPUT_PLACEHOLDER);
+  });
+
+  it("shoud be works with EditableText mode number", () => {
```
Comment (tritonJS826) [inline mw-webapp/src/component/editableText/EditableText.cy.tsx:65]:
```suggestion
  it("shoud work with mode 'number'", () => {
```

File: mw-webapp/src/component/editableText/EditableText.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      value={props.value}
+      onChangeFinish={() => {}}
+      isEditable={true}
+      type={props.type ?? InputType.Text}
+    />
+  );
+};
+
+describe("EditableText component", () => {
+  it("shoud be renders when placeholder is visible", () => {
```
Comment (tritonJS826) [inline mw-webapp/src/component/editableText/EditableText.cy.tsx:58]:
```suggestion
  it("shoud work when placeholder is visible", () => {
```

**Review by tritonJS826 — COMMENTED** [review]
Lets also check that onFinishChange triggered, and that we could type something

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/component/editableText/EditableText.cy.tsx (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/component/editableText/EditableText.cy.tsx]: Done

File: mw-webapp/src/component/editableText/EditableText.cy.tsx (hunk 2 above)
Comment (Alex89198900) [reply mw-webapp/src/component/editableText/EditableText.cy.tsx]: Done

File: mw-webapp/src/component/editableText/EditableText.cy.tsx (hunk 3 above)
Comment (Alex89198900) [reply mw-webapp/src/component/editableText/EditableText.cy.tsx:58]: Done

**Pull request merged**